frontend/lib/src/components/shared/FullScreenWrapper/FullScreenWrapper.tsx

```diff
@@ -143,13 +143,16 @@ class FullScreenWrapper extends PureComponent<Props, State> {
 
     return (
       <StyledFullScreenFrame isExpanded={expanded}>
-        <StyledFullScreenButton
-          onClick={buttonOnClick}
-          title={buttonTitle}
-          isExpanded={expanded}
-        >
-          <Icon content={buttonImage} />
-        </StyledFullScreenButton>
+        {!this.context.hideFullScreenButton && (
```

Comment:
Nit: Think we should destructure like we do for props/state above:
`const { hideFullScreenButton } = this.context`
and then just use `!hideFullScreenButton` here